shell/src/main/java/org/apache/accumulo/shell/Shell.java

```diff
@@ -618,7 +622,7 @@ public int start() throws IOException {
         writer.println();
 
         String partialLine = uie.getPartialLine();
-        if (partialLine == null || "".equals(uie.getPartialLine().trim())) {
+        if (partialLine == null || uie.getPartialLine().trim().isEmpty()) {
```

Comment:
`uie.getPartialLine()` was called on the prior line. I think you can just change this to `partialLine.trim().isEmpty()`